### PR TITLE
Add r-survtmle

### PR DESCRIPTION
**Pull request by leej3**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by leej3** [comment]
@conda-forge/help-r

File: recipes/r-survtmle/LICENSE
Comment (mfansler) [inline recipes/r-survtmle/LICENSE:1]: This file can be removed - it will come with the tarball.

File: recipes/r-survtmle/build.sh
Comment (mfansler) [inline recipes/r-survtmle/build.sh:3]: Remove comments.

File: recipes/r-survtmle/build.sh
Comment (mfansler) [inline recipes/r-survtmle/build.sh:13]: The `Priority` thing is not considered an issue anymore. Remove.

File: recipes/r-survtmle/meta.yaml
Comment (mfansler) [inline recipes/r-survtmle/meta.yaml:20]: Should be a `noarch` build. Add

```yaml
  noarch: generic
```

File: recipes/r-survtmle/meta.yaml
Comment (mfansler) [inline recipes/r-survtmle/meta.yaml:18]: Remove comments.

File: recipes/r-survtmle/meta.yaml
Comment (mfansler) [inline recipes/r-survtmle/meta.yaml:73]: All R packages should include `conda-forge/r` as a maintainer.

File: recipes/r-survtmle/meta.yaml
Comment (mfansler) [inline recipes/r-survtmle/meta.yaml:30]: Be prepared for cross-compiled builds by including:

```yaml
    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for the submission. It appears an older recipe generator was used. All suggested changes would be fixed if the latest [`conda_r_skeleton_helper`](https://github.com/bgruening/conda_r_skeleton_helper) were used instead.

**Comment by leej3** [comment]
Thanks for the tips. I will use that skeleton helper in the future. Fixes applied...

Regarding `noarch: generic` I was unsure because there is byte-code compiled as part of the package creation. I couldn't find extensive docs to describe how one decides upon a packages noarch status. Are there any docs you could point me to on this? Similarly, I can't find any docs regarding support of cross-compilation...

**Review by mfansler — APPROVED** [review]

**Comment by mfansler** [comment]
[The DESCRIPTION file](https://github.com/cran/survtmle/blob/295de8b4be182bc6696afd2ef06f0f738d3a5eae/DESCRIPTION#L28) reports no compilation.

There are some sections in the Knowledge Base that pertain to targeting additional platforms, e.g., [**osx-arm64**](https://conda-forge.org/docs/maintainer/knowledge_base.html#apple-silicon-builds), [**linux-aarch64**](https://conda-forge.org/docs/maintainer/knowledge_base.html#using-arch-rebuild-txt). Admittedly, they don't describe in detail the cross-compilation strategy. For working purposes, new recipes in `staged-recipes` should cover just the default supported platforms. After merging and feedstock creation, the package can be added to the desired migration list and a migration bot will create a PR that converts the recipe to support the requested platform. So, it's mostly automated.

**Comment by mfansler** [comment]
@conda-forge/staged-recipes looks good to me

**Pull request merged**
